pandas/tools/plotting.py

```diff
@@ -2350,10 +2359,14 @@ def _plot(data, x=None, y=None, subplots=False,
 df_ax = """ax : matplotlib axes object, default None
     subplots : boolean, default False
         Make separate subplots for each column
-    sharex : boolean, default True
-        In case subplots=True, share x axis
+    sharex : boolean, default True if ax is None else False
+        In case subplots=True, share x axis and set some x axis labels to
+        invisible; defaults to True is ax is None or False if an ax is
```

Comment:
Typo: "is" should be "if" here after the True. Maybe change the "or" to "otherwise"